blind_75/reorder_list.py

```diff
@@ -0,0 +1,74 @@
+# https://leetcode.com/problems/reorder-list/ 
```

Comment:
Note: This algo doesn't work and compile. I spent 50 minutes or so but got stuck and started doing linked list problems

Reply:
I also didn't do pseudocode here